Formula/a/apify-cli.rb

```diff
@@ -0,0 +1,27 @@
+require "language/node"
+
+class ApifyCli < Formula
+  desc "Apify command-line interface"
+  homepage "https://docs.apify.com/cli"
+  url "https://registry.npmjs.org/apify-cli/-/apify-cli-0.18.0.tgz"
+  sha256 "28d4e8b032992560a4f842a44e8d107693b1a031560325b90967245c4ed7b628"
+  license "Apache-2.0"
+
+  depends_on "node@16"
```

Comment:
Does it work with `node` 20 (or if not, `node@18`)? `node@16` [will reach its EOL on 2023-09-11](https://github.com/nodejs/release#release-schedule), (i.e., in 2 weeks).